doc/reference/configuration/configuration_reference.rst

```diff
@@ -1680,6 +1680,135 @@ The ``groups`` section provides the ability to define the :ref:`full topology of
 
 
+..  _configuration_reference_memtx:
+
+memtx
+-----
+
+The ``memtx`` section is used to configure parameters related to :ref:`memtx engine <configuration_memtx>`.
+
+.. NOTE::
+
+    ``memtx`` can be defined in any :ref:`scope <configuration_scopes>`.
+
+
+-   :ref:`memtx.allocator <configuration_reference_memtx_allocator>`
+-   :ref:`memtx.max_tuple_size <configuration_reference_memtx_max_size>`
+-   :ref:`memtx.memory <configuration_reference_memtx_memory>`
+-   :ref:`memtx.min_tuple_size <configuration_reference_memtx_min_size>`
+-   :ref:`memtx.slab_alloc_factor <configuration_reference_memtx_slab_alloc_factor>`
+-   :ref:`memtx.slab_alloc_granularity <configuration_reference_memtx_slab_alloc_granularity>`
+-   :ref:`memtx.sort_threads <configuration_reference_memtx_sort_threads>`
+
+..  _configuration_reference_memtx_allocator:
+
+..  confval:: memtx.allocator
+
+    Specify the allocator used for memtx tuples.
+    The possible values are ``system``  and ``small``:
+
+    *   ``system`` is based on the ``malloc`` function.
+        The allocator allocates memory as needed, checking that the quota is not exceeded.
+
+    *   ``small`` is a special `slab allocator <https://github.com/tarantool/small>`_.
+        The allocator repeatedly uses a memory block to allocate objects of the same type.
+        Note that this allocator is prone to unresolvable fragmentation on specific workloads,
+        so you can switch to ``system`` in such cases.
+
+    |
+    | Type: string
+    | Default: 'small'
+    | Environment variable: TT_MEMTX_ALLOCATOR
+
+..  _configuration_reference_memtx_max_size:
+
+..  confval:: memtx.max_tuple_size
+
+    Size of the largest allocation unit for the memtx storage engine in bytes.
+    It can be increased if it is necessary to store large tuples.
+    See also: ``vinyl.max_tuple_size``.
+
+    |
+    | Type: integer
+    | Default: 1048576
+    | Environment variable: TT_MEMTX_MAX_TUPLE_SIZE
+
+..  _configuration_reference_memtx_memory:
+
+..  confval:: memtx.memory
+
+    The amount of memory in bytes that Tarantool allocates to actually store tuples.
+    When the limit is reached, :ref:`INSERT <box_space-insert>` and
+    :ref:`UPDATE <box_space-insert>` requests fail with the  :errcode:`ER_MEMORY_ISSUE` error.
+    The server does not go beyond the ``memtx.memory`` limit to allocate tuples, but there is additional memory
+    used to store indexes and connection information.
+
+    |
+    | Type: integer
+    | Default: 268435456
+    | Environment variable: TT_MEMTX_MEMORY
+
+..  _configuration_reference_memtx_min_size:
+
+..  confval:: memtx.min_tuple_size
+
+    Size of the smallest allocation unit in bytes.
+    It can be decreased if most of the tuples are very small.
+    The value must be between 8 and 1048280 inclusive.
+
+    |
+    | Type: integer
+    | Default: 16
+    | Environment variable: TT_MEMTX_MIN_TUPLE_SIZE
```

Comment:
```suggestion
    It can be decreased if most of the tuples are very small.

    |
    | Type: integer
    | Default: 16
    | Possible values: between 8 and 1048280 inclusive
    | Environment variable: TT_MEMTX_MIN_TUPLE_SIZE
```